### PR TITLE
I have submitted my week 5 site deweycheetemandhowe.html

### PR DESCRIPTION
**Pull request by JennHopkins**
Here is the best I can do for the things requested of me.

Also, updated EZbreezy (and adoptadog).

See, I'm good for something.

### CODE REVIEW

File: deweycheetemandhowe.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      <p>Click the button to:
+      <ul>
+        <li>make a Withdraw (if you must)</li>
+        <li>give a Deposit (gimmie, gimmie, gimmie)</li>
+        <li>See your balance (lookit all them bananas!)</li>
+        <li>Or Quit (while you're ahead)</li>
+      </ul>
+      <button onclick = "bankAction()">Sucker!</button>
+      <p>Hello, Bankee, what would you like D,C and H Money Banks do to you today?</p>
+    <script>
```
Comment (jimmynono) [inline deweycheetemandhowe.html]: You'll always want to do all your JS work in JS files, not inline in your HTML

File: deweycheetemandhowe.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+   function withdrawing() {
+     let input = prompt('Enter Withdraw amount below')
+     let totalBalance = 500;
+     let result = 0;
+
+    result = totalBalance - Number(input);
+    alert('Your new balance is now' + '$' + result);
+   }
+
+   function depositing() {
```
Comment (jimmynono) [inline deweycheetemandhowe.html]: Each time you call this function it resets the balance to 500 (the withdrawl too)

**Comment by JennHopkins** [comment]
Okay, I have re-done the bank app - still not happy with the "balance" portion of it (gonna ask my friend about that), but everything works exactly as it's supposed to.

Here you go!

**Comment by JennHopkins** [comment]
By the way - I was surprised to learn about the "while loop".  Nobody told me that a "while loop" could have a comparison condition. EVERY single example I found online and even in this class made it look like it could only do a numerical value for its condition statement. I thought that it's proper syntax was: while(variable < 3) { //do this code}; Because every single example had the greater than (or less than) symbol and then a number value. I did experiment with an equal sign operator: while(variable = d) {//do this code}; but it didn't make anything happen, I therefore assumed (incorrectly) that it wouldn't work without the greater than (or less than) symbol and a number - but I had no other source of information to tell me otherwise. Well, now I know.

**Comment by JennHopkins** [comment]
Okay, so I have fixed the balance function so that it will show you how much you have either withdrawn or deposited. I have also added a transfer function (though you have to blame my friend Roxanne for that, she talked me into it). Oh Roxy, look what you made me do!

**Pull request closed without merging**